src/introduction.md

```diff
@@ -5,3 +5,192 @@ but try to keep to the same general structure.
 
 If there's a timeline associated with the area you're describing,
 put that in a `##` section at the end of the page.
+
+
+## Logistics
+
+### Rooms
+- good to have space outside the rooms, to do chaotic things and self organize
+- the downstairs is out of the way, gives more freedom. Ballrooms might provide more space
+- Conference Center: we got people looking for SCALE registration
+- Not set up specifically for workshop-style
+
+### AV:
+- needed lights dimmed
+- screen size not enough for size of room
+- talks had a small popup
+- Can we pre-arrange for better AV setup?
+- tell presenters to be prepared with "dark/light mode"
+- coach speakers on mic technique.
+  - Preparation time, or coordination meeting
+- number of mics: ask up front how they will choreograph
+  - can we ask SCALE for 3 mics? easier for Q+A
+- SPEAKERS need to repeat question
+  - have people get up, stand in line, and ask questions
+  - tight rows discourages this
```

Comment:
Maybe have someone introduce the next speaker? I saw that at other conferences and it adds a professional touch.